### DISCUSSION TITLE
Dynamic Int option is not showing in toggle button

### DISCUSSION THREAD

**Discussion by VRbeast** [micjahn/ZXing.Net#542] [General]
![Screenshot 2023-12-20 003119](https://github.com/micjahn/ZXing.Net/assets/154358777/145a2876-6ba6-4c27-9071-acd5a85fd198)
![Screenshot 2023-12-20 003216](https://github.com/micjahn/ZXing.Net/assets/154358777/579eeb88-e5bf-4f06-8936-58285cd7658b)

Hi my dynamic int option is now showing at the toggle button. If there anyone has the solution pls comment down

**Comment by micjahn** [comment]
Wrong discussion forum. This question has no relation to the ZXing.Net project.
